rules/okta/okta_attempt_to_modify_or_delete_application_sign_on_policy.toml

```diff
@@ -0,0 +1,35 @@
+[metadata]
+creation_date = "2020/07/01"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+An adversary may attempt to modify or delete the sign on policy for an Okta application in order to remove or weaken an
+organization's security controls.
+"""
+false_positives = [
+    """
+    Consider adding exceptions to this rule to filter false positives if sign on policies for Okta applications are
+    regularly modified or deleted in your organization.
+    """,
+]
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Modification or Removal of an Okta Application Sign On Policy"
```

Comment:
```suggestion
name = "Modification or Removal of an Okta Application Sign-On Policy"
```
?

Reply:
from https://developer.okta.com/docs/reference/api/event-types/
> **application.policy.sign_on.rule.delete**
> Legacy event types: app.app_instance.sign_on_policy.delete_rule
> Delete rule from app sign on policy.

I guess that's how they spell it? 🤷

Reply:
So long as we consistently use it across all rules, I am good with either 👍

Reply:
I'll change it to `Sign-On`. Looks like none of our other rules contain this term yet.